Makefile

```diff
@@ -150,3 +147,20 @@ node_modules/.uptodate: package.json
 	@echo installing javascript dependencies
 	@node_modules/.bin/check-dependencies 2>/dev/null || npm install
 	@touch $@
+
+PYTHON_VERSION = 3.6.8
+PYTHON_MINOR_VERSION := $(shell echo $$PYTHON_VERSION | cut -d '.' -f1,2)
+PYTHON_VERSION_INSTALLED := $(shell python$${PYTHON_MINOR_VERSION} --version 2>/dev/null)
```

Comment:
This is slightly unfortunate but:

1. `PYTHON_VERSION` is the one string that we need to change when we want to upgrade the version of Python. It's needed for the `pyenv install 3.6.8` command (`pyenv install 3.6` doesn't work)
2. `PYTHON_MINOR_VERSION` is `3.6` (if `PYTHON_VERSION` is `3.6.x`). It's need for the `python3.6` command (there is no `python3.6.8` executable)
3. `PYTHON_VERSION_INSTALLED` is a variable that is defined if Python 3.6 (any patch version of 3.6) is installed, undefined otherwise. It's used to decide whether `pyenv install 3.6.8` needs to be run or not.